dot/network/status.go

```diff
@@ -193,10 +199,13 @@ func (status *status) expireStatus(ctx context.Context, peer peer.ID) {
 	time.Sleep(ExpireStatusInterval)
 
 	// get time of last confirmed status message
-	lastConfirmed := status.peerConfirmed[peer]
+	lastConfirmed, ok := status.peerConfirmed.Load(peer)
```

Comment:
this could be simplified in a oneliner.

Reply:
same as above, `lastConfirmed` is used again at line 208